etna/metrics/metrics.py

```diff
@@ -178,4 +178,28 @@ def __init__(self, mode: str = MetricAggregationMode.per_segment, **kwargs):
         super().__init__(mode=mode, metric_fn=msle, **kwargs)
 
 
-__all__ = ["MAE", "MSE", "R2", "MSLE", "MAPE", "SMAPE", "MedAE"]
+class Sign(Metric):
+    """Sign metric.
+
+    ..math::
+        Sign = \\frac{\\sum_{i=0}^{n-1}{sign(y\_true_i - y\_pred_i)}}{n}
```

Comment:
https://deploy-preview-730--etna-docs.netlify.app/api/etna.metrics.metrics.sign#etna.metrics.metrics.Sign
Not rendering correcltly
And you should add parameters to tex formula like in other places `Sign(y_true, y_pred)`

Reply:
There is rendering problem with `metric.functional` too